impl/githubV4.js

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env node
+
+'use strict';
+
+(() => {
```

Comment:
FYI, you don't need these `(() => {` and `})();`: A variable created in a Node.js module is already private